spec/meilisearch/client/multi_search_spec.rb

```diff
@@ -16,4 +16,19 @@
     expect(response['results'][0]['estimatedTotalHits']).to eq(0)
     expect(response['results'][1]['estimatedTotalHits']).to eq(0)
   end
+
+  it 'does a federated search with two different indexes' do
```

Comment:
Does that meet your expectations?

Reply:
Actually, from what I understood here @Nymuxyzo you would need to make two tests:

- one asserting the inexistence of the presence of the param federation in the response
- one asserting that you explicitly called the `multi_search` with a federation request, since the doc says a non-null value the value could be `multi_search(data, federation: true, federation_options: {....}) # where federation options are not required`

https://www.meilisearch.com/docs/learn/multi_search/multi_search_vs_federated_search#what-is-federated-search